lib/ClangImporter/ClangImporter.cpp

```diff
@@ -6525,6 +6525,7 @@ CxxRecordSemanticsKind
 CxxRecordSemantics::evaluate(Evaluator &evaluator,
                              CxxRecordSemanticsDescriptor desc) const {
   const auto *decl = desc.decl;
+  auto &clangSema = desc.ctx.getClangModuleLoader()->getClangSema();
```

Comment:
Is this just for side-effects? I don't see it getting used.

Reply:
There are no side effects here. This line can be removed.